docker-compose.yml

```diff
@@ -8,6 +8,17 @@ services:
     volumes:
       - .:/go/src/github.com/m3db/m3
       - /usr/bin/buildkite-agent:/usr/bin/buildkite-agent
+      # Support running docker within docker. That is, buildkite jobs themselves run in a container; that container
+      # needs to be able to spin up functioning docker containers.
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
![We need to go deeper](https://i.kym-cdn.com/photos/images/newsfeed/000/531/557/a88.jpg)